src/api/todos.ts

```diff
@@ -1,8 +1,33 @@
 import { Todo } from '../types/Todo';
 import { client } from '../utils/fetchClient';
 
-export const getTodos = (userId: number) => {
+export const getTodos = (userId?: number) => {
   return client.get<Todo[]>(`/todos?userId=${userId}`);
 };
 
-// Add more methods here
+export const createTodo = (title: string, userId?: number) => {
+  return client.post<Todo>('/todos', {
+    userId,
+    completed: false,
+    title,
+  });
+};
+
+export const updateTodo = (todoId?: number, value?: boolean) => {
+  return client.patch<Todo>(`/todos/${todoId}`, {
+    completed: value,
+  });
+};
```

Comment:
Same for these functions. In my opinion all params are required here 
```suggestion
export const createTodo = (title: string, userId: number) => {
  return client.post<Todo>('/todos', {
    userId,
    completed: false,
    title,
  });
};

export const updateTodo = (todoId: number, value: boolean) => {
  return client.patch<Todo>(`/todos/${todoId}`, {
    completed: value,
  });
};
```